util/Migrator/DbScripts/2025-05-20_00_AddSendEmails.sql

```diff
@@ -0,0 +1,128 @@
+-- Add `Emails` field that stores a comma-separated list of email addresses for
+-- email/OTP authentication to table and write methods. The read methods
+-- don't need to be updated because they all use `*`.
+ALTER TABLE [dbo].[Organization] ADD [Emails] NVARCHAR(1024) NULL;
```

Comment:
❓ Why 1024?

Reply:
It's an arbitrary number that was accepted as a reasonable max length by product/design. 1 kb is large enough to fit 10-20 typical email addresses (4 max-length email addresses), which seems an adequate length for an MVP. If we need more than that, it's a good indicator that we need to switch to a document paradigm.

On that last point: I wanted to switch to a document paradigm during email/OTP, but there's open questions about how to support it.